runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkBatchPortablePipelineTranslator.java

```diff
@@ -595,4 +624,15 @@ private static void pruneOutput(
             String.format("%s/out.%d", transformName, unionTag));
     context.addDataSet(collectionId, pruningOperator);
   }
+
+  private static <T> Coder<WindowedValue<T>> instantiateCoder(
```

Comment:
Move this to `FlinkPipelineTranslatorUtils` ?

Reply:
Moved.